third_party/2and3/decorator.pyi

```diff
@@ -0,0 +1,85 @@
+import sys
+from typing import Any, Callable, Dict, List, NamedTuple, Optional, Pattern, Text, Tuple, TypeVar
+
+_C = TypeVar("_C", bound=Callable[..., Any])
+_Func = TypeVar("_Func", bound=Callable[..., Any])
+
+def get_init(cls): ...
+
+# Duplicated from `stdlib/3/inspect.pyi`
+FullArgSpec = NamedTuple('FullArgSpec', [('args', List[str]),
+                                         ('varargs', Optional[str]),
+                                         ('varkw', Optional[str]),
+                                         ('defaults', tuple),
```

Comment:
`tuple` -> `Tuple[Any, ...]`. We now run mypy with `--disallow-any-generics`, which would flag this.